book/src/java_object.md

```diff
@@ -0,0 +1 @@
+# The `JavaObject` trait
```

Comment:
Why this empty page? Is it linked from elsewhere in the book?

Reply:
I think this trait was going to have a bigger role than it would up having...? Probably an ok page to have though.